site/content/in-dev/unreleased/metastores.md

```diff
@@ -32,6 +32,29 @@ In order to add other JDBC drivers, you have to build Polaris using the `eclipse
 
 ## Polaris Server Configuration
 
+### Relational JDBC
+Configure the `polaris.persistence` section in your Polaris configuration file
+(`application.properties`) as follows:
+
+```
+polaris.persistence.type=relational-jdbc
```

Comment:
Is it OK to miss as `relational-jdbc` should be the default value now?

Reply:
I think being explicit is helpful just in case we change the default in future ? wdyt ?